windows/bleachbit.nsi

```diff
@@ -355,28 +355,30 @@ Section Core (Required)
         "URLInfoAbout" "https://www.bleachbit.org/"
     WriteRegStr SHCTX "${MULTIUSER_INSTALLMODE_UNINSTALL_REGISTRY_KEY_PATH}$0" \
         "URLUpdateInfo" "https://www.bleachbit.org/download"
-
+    WriteRegStr HKLM "${MULTIUSER_INSTALLMODE_UNINSTALL_REGISTRY_KEY_PATH}$0" \
```

Comment:
Instead of using HKLM it is better to use SHCTX, as it will be replaced appropriately with HKLM or HKCU at runtime according to SetShellVarContext